Extension/src/LanguageServer/timeTelemetryCollector.ts

```diff
@@ -0,0 +1,84 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+import * as telemetry from '../telemetry';
+import * as util from '../common';
+import * as vscode from 'vscode';
+
+interface TimeStampSequence {
+    firstFile?: number; // when the extension is activated by realActivation. Defined only for "cold" start cases.
+    didOpen: number; // when the file appears in the editor. Defined for both "warm" start cases.
```

Comment:
> when the file appears in the editor. Defined for both "warm" start cases.

What are the two warm start cases? Or does this comment mean to say "for both warm and cold start cases"?

Reply:
"both" needs to be deleted